src/Microsoft.Management.Deployment/PackageManager.idl

```diff
@@ -1214,6 +1214,206 @@ namespace Microsoft.Management.Deployment
         String InstallationNotes { get; };
     }
 
+    /// IMPLEMENTATION NOTE: AddPackageCatalogOptions
+    [contract(Microsoft.Management.Deployment.WindowsPackageManagerContract, 11)]
+    runtimeclass AddPackageCatalogOptions
+    {
+        AddPackageCatalogOptions();
+
+        /// The name of the catalog to add.
+        String CatalogName;
+        /// The source URI of the catalog to add.
+        String SourceUri;

```

Comment:
Technically this is an all-purpose string value that the code for the `Type` can interpret however it wants.  I get that this becomes awkward and requires more documentation on an API though.  The value will be returned through the `String Argument` property of `PackageCatalogInfo` though, so I think it makes sense to remain a string here.